i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md

```diff
@@ -3,6 +3,18 @@ id: cloud-release-notes
 title: Centreon Cloud - notes de release
 ---
 
+## December 17th, 2024
+
+This release contrains numerous bugs and vulnerability fixes but also the following improvements:
+
+### Centreon IT Edition
+
+**Data Collection**
+
+- Thanks to Additional Connector Configurations feature, you can now define the credentials to access a monitored resource through the Centreon interface rather than having to connect to your pollers manually.
```

Comment:
```suggestion
- Thanks to the Additional Connector Configurations feature, you can now define the credentials to access a monitored resource through the Centreon interface rather than having to connect to your pollers manually.
```